core/src/main/java/com/sequenceiq/distrox/v1/distrox/converter/InstanceTemplateParameterConverter.java

```diff
@@ -80,11 +83,15 @@ private void initAzureEncryptionFromEnvironment(AzureInstanceTemplateV4Parameter
                 .map(AzureEnvironmentParameters::getResourceEncryptionParameters)
                 .map(AzureResourceEncryptionParameters::getDiskEncryptionSetId)
                 .orElse(null);
+        String environmentName = environment.getName();
         if (diskEncryptionSetId != null) {
+            LOGGER.info("Applying SSE with CMK for Azure managed disks as per environment \"{}\".", environmentName);
```

Comment:
shouldn't the log context contain every necessary information and env name is just duplication of data here?

Reply:
Good point, will remove the msg parameter.

Reply:
Done.